### PR TITLE
Add integration workflow

### PR DESCRIPTION
**Pull request by quentinleroy**

### CODE REVIEW

File: source/includes/_errors.md [hunk 1]
```diff
@@ -54,12 +54,14 @@ Error Code  | Meaning
 400.101.038 | Extra quantity is invalid or missing
 400.101.039 | Marketing carrier booking number is invalid or missing
 400.101.040 | Marketing carrier is invalid or missing
-400.100.041 | Booking is invalid or missing
+400.101.041 | Booking is invalid or missing
```
Comment (dasnotme) [inline source/includes/_errors.md]: - [ ] This might be a breaking change.

Comment (dasnotme) [reply source/includes/_errors.md]: To make it non-breaking, we can keep both, but not to use `400.100.041` in the code.

File: source/includes/_introduction.md [hunk 2]
```diff
@@ -33,6 +33,17 @@ When performing a booking through Distribusion API, there are multiple steps tha
 
 ![alt text](/images/API_V3.0_workflow.png)
 
+## Integration Workflow
+
+1. _(1-4 weeks)_ **Integrate our API**, using this documenation. 
+If you have any questions, do not hesitate to write me directly at [quentin.leroy@distribusion.com](mailto:quentin.leroy@distribusion.com)
```
Comment (dasnotme) [inline source/includes/_introduction.md]: - [ ] You might get some spam-bots attracted ;)

File: source/includes/_errors.md (hunk 1 above)
Comment (quentinleroy) [reply source/includes/_errors.md]: It was just a typo when writing it. The actual code is 400.101.041

File: source/includes/_introduction.md (hunk 2 above)
Comment (quentinleroy) [reply source/includes/_introduction.md]: Good point :)

**Pull request merged**